src/main/cpp/t06_quad.cpp

```diff
@@ -21,5 +21,17 @@
 using namespace std;
 
 int t06_quad() {
-
+    double a, b, c, D, x1, x2, x0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x2' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4716804)